### PR TITLE
Remove @emotion/styled

### PR DESCRIPTION
**Pull request by nring**
Hopefully this helps! I'll leave a few inline comments.

I didn't change versioning as I'll leave that up to you. But following semver I'd say this is `1.6.0`

### CODE REVIEW

File: src/components/Field/index.js [hunk 1]
```diff
@@ -1,13 +1,47 @@
...
+            {label && <div style={labelStyles}>{label}</div>}
+            <div className="storybook-addon-mock-fieldItem" style={fieldItemStyles}>
+                <style>{`
+                    .storybook-addon-mock-fieldItem > div > div > svg > path {
+                        opacity: 0;
+                    }
+                    .storybook-addon-mock-fieldItem > div > div > div > span > div > div > svg > path {
+                        opacity: 0;
+                    }
+                `}</style>
```
Comment (nring) [inline src/components/Field/index.js:39]: I'm not proud of this, but it seems to do the trick.

Comment (nutboltu) [reply src/components/Field/index.js:39]: We may create a separate react component naming `GlobalStyles`

```
export const GlobalStyles = () => (
  <style>{`
        .storybook-addon-mock-fieldItem > div > div > svg > path {
            opacity: 0;
        }
        .storybook-addon-mock-fieldItem > div > div > div > span > div > div > svg > path {
            opacity: 0;
        }
    `}
  </style>
);
```
and import it in the parent component `RequestItem`.

```
export const RequestItem = () => {
  ...
  return (
      <GlobalStyles />
     ...
  );
}
```

**Comment by nring** [comment]
Updated PR based off your comments. Another thing I was thinking is to change the style to be less dependent on DOM structure. The first item could be rewritten as
```css
        .storybook-addon-mock-fieldItem path[fill="green"] {
            opacity: 0;
        }
```

Though I'm not sure what svg you were trying to target with the second item.

**Comment by nutboltu** [comment]
@nring  this is the cleaner solution if it works. These pieces of CSS are specific to the `JSONInput` component. The first one is for the success SVG and the second one is for the error.

**Review by nutboltu — APPROVED** [review]
@nring this PR looks good to me. I am going to merge this PR. we can improve the svg targeted CSS in a later PR.

**Pull request merged**